### PR TITLE
Added table syntax

### PR DESCRIPTION
**Pull request by adilJamshad**

### CODE REVIEW

**Comment by malivinayak** [comment]
@adilJamshad 

To add a syntax which can visible in markup code you have to use triple back tick sign i.e. ``` 

e.g.
```
| Sr  | Name |
| --- | ---- |
| 0   | Adil |
```

File: README.md [hunk 1]
```diff
@@ -45,3 +43,7 @@
 ## Text Style
 
 ## Table
+
```
Comment (malivinayak) [inline README.md:46]: add ``` here

File: README.md [hunk 2]
```diff
@@ -45,3 +43,7 @@
...
+| Sr  | Name |
+| --- | ---- |
+| 0   | Adil |
```
Comment (malivinayak) [inline README.md:49]: add ``` here below this line

**Review by malivinayak — CHANGES_REQUESTED** [review]
add ``` triple back tick so that others can see markup code directly

**Comment by malivinayak** [comment]
Thank You for your contribution @adilJamshad 
Just make that two small changes and I will merge your PR

**Comment by malivinayak** [comment]
@adilJamshad just comment in #3 so that I can assign your for that issue

**Comment by malivinayak** [comment]
Refer #7  To make PR

File: README.md (hunk 2 above)
Comment (adilJamshad) [reply README.md:50]: I have added ``` here

File: README.md (hunk 1 above)
Comment (adilJamshad) [reply README.md:46]: I have added ``` here

**Comment by malivinayak** [comment]
In order to make it more user friendly you have to follow this structure
image

**Pull request closed without merging**